app/models/viewModels/PastAftReturnsViewModel.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.viewModels
+
+import models.PastAftReturnGroup
+import play.api.libs.json.{Json, OWrites}
+
+case class PastAftReturnsViewModel(aftReturnGroup: List[PastAftReturnGroup])
+
+object PastAftReturnsViewModel {
+  implicit lazy val writes: OWrites[PastAftReturnsViewModel] =
+    Json.writes[PastAftReturnsViewModel]
+}
```

Comment:
Why is this necessary?
Why not just use List[PastAftReturnGroup] instead of this class?

Reply:
Seems like a nunjucks requirement. No need to change.